diff --git a/lib/tsserver.js b/lib/tsserver.js
index 63bcdac309ff42bedada2d018b795be57ee2fe8d..cd9dd67c5a593b9dc721b3d861eb178227ddd6a6 100644
--- a/lib/tsserver.js
+++ b/lib/tsserver.js
@@ -14987,7 +14987,7 @@ function isRestParameter(node) {
 // src/compiler/utilities.ts
 var resolvingEmptyArray = [];
 var externalHelpersModuleNameText = "tslib";
-var defaultMaximumTruncationLength = 160;
+var defaultMaximumTruncationLength = 1e6;
 var noTruncationMaximumTruncationLength = 1e6;
 function getDeclarationOfKind(symbol, kind) {
   const declarations = symbol.declarations;