diff --git a/lib/tsserver.js b/lib/tsserver.js
index b696263504d6854ad0f889b21eeed22d00500cd1..0cfcce355f501e02c5bd74f2eba2a915df92d051 100644
--- a/lib/tsserver.js
+++ b/lib/tsserver.js
@@ -14335,7 +14335,7 @@ var ts;
 (function (ts) {
     ts.resolvingEmptyArray = [];
     ts.externalHelpersModuleNameText = "tslib";
-    ts.defaultMaximumTruncationLength = 160;
+    ts.defaultMaximumTruncationLength = 1000000;
     ts.noTruncationMaximumTruncationLength = 1000000;
     function getDeclarationOfKind(symbol, kind) {
         var declarations = symbol.declarations;