diff --git a/lib/tsserver.js b/lib/tsserver.js
index 6158bf4eb0a407e9598e128a2f2cd1615f3373d4..b52d0d06dadc8697989a21b766373a2468208a8b 100644
--- a/lib/tsserver.js
+++ b/lib/tsserver.js
@@ -14920,7 +14920,7 @@ function isRestParameter(node) {
 // src/compiler/utilities.ts
 var resolvingEmptyArray = [];
 var externalHelpersModuleNameText = "tslib";
-var defaultMaximumTruncationLength = 160;
+var defaultMaximumTruncationLength = 1e6;
 var noTruncationMaximumTruncationLength = 1e6;
 function getDeclarationOfKind(symbol, kind) {
   const declarations = symbol.declarations;